README.md

```diff
@@ -70,6 +70,6 @@ In addition to raw reactive streams support, the `netty-reactive-streams-http` l
 
 ## Support
 
-The Play Mailer library is *[Community Driven][]*.
+The Netty Reactive Streams library is *[Supported][]*.
```

Comment:
I think this was a copy/paste error. According to https://developer.lightbend.com/docs/lightbend-platform/introduction/getting-help/build-dependencies.html#_netty_reactive_streams (which was copied from the previous documentation) this library is supported. Does that seem right to you @marcospereira?

Reply:
Yeah, that seems right to me.